data/interim/mesh_pipeline_result.csv.dvc

```diff
@@ -0,0 +1,4 @@
+outs:
+- md5: 10639bbe244b986919efc8f7866b98b4
+  size: 138862818
+  path: mesh_pipeline_result.csv
```

Comment:
did you add that manually?

Reply:
I think this is a mistake, I will have a look

Reply:
No that is correct. This is generated by `trigger_pipeline.sh` which depends on an external repository, so at the moment I am generating it manually.

Reply:
And are those grants? Maybe the name could be more descriptive but I understand this might require you to rerun. Anyway if you do maybe `mesh_pipeline_tagged_grants.csv`